app/views/shared/_nav_bar.html.erb

```diff
@@ -0,0 +1,109 @@
+                                    <i class="fas fa-network-wired"></i> Organizations
+                                <% end %>
+                            <% end %>
+                            <% if current_admin.accessible_organizations(:manage).any? || current_admin.power_user? %>
+                                <%= link_to (admin_protocols_path), class: "dropdown-item #{active_controller?("admin/protocols")}" do %>
+                                    <i class="fas fa-prescription-bottle-alt"></i> Protocol medications
+                                <% end %>
+                            <% end %>
+                            <% if current_admin.accessible_facilities(:manage).any? || current_admin.power_user? %>
+                                <%= link_to (admins_path), class: "dropdown-item #{active_controller?("admins")}" do %>
+                                    <i class="fas fa-user-friends"></i> Dashboard admins
+                                <% end %>
+                            <% end %>
+                            <% if current_admin.accessible_users(:manage).any? || current_admin.power_user? %>
+                                <%= link_to (admin_users_path), class: "dropdown-item #{active_controller?("admin/users")}", id: "mobile-app-users" do %>
+                                    <i class="fas fa-user-md"></i> Mobile app users
+                                <% end %> 
+                            <% end %>
+                            <% if current_admin.power_user? %>
+                                <%= link_to ("/sidekiq"), class: "dropdown-item #{active_controller?("sidekiq")}" do %>
+                                    <i class="fas fa-running"></i> Sidekiq
+                                <% end %>
+                            <% end %>
+                            <% if current_admin.power_user? %>
+                                <%= link_to ("/flipper"), class: "dropdown-item #{active_controller?("flipper")}" do %>
+                                    <i class="fas fa-circle-notch"></i> Flipper
+                                <% end %> 
+                            <% end %>
+                        <% end %>
+                        <div class="dropdown-divider"></div>
+                        <h6 class="dropdown-header">
+                            Resources
+                        </h6>
+                        <%= link_to (resources_path), class: "dropdown-item #{active_controller?("resources")}" do %>
+                            <i class="fas fa-chalkboard-teacher"></i> Training materials
+                        <% end %>
+                        <div class="dropdown-divider"></div>
+                        <h6 class="dropdown-header">
+                            Legal
+                        </h6>
+                        <a href="https://www.simple.org/privacy/" class="dropdown-item">
```

Comment:
The icons indicate these docs will open in a different tab/window. Could we add a `target=_blank` to the Privacy Policy and License links?

Reply:
Good call!